### ISSUE TITLE
updating Java blocked by absence of packaging tool

### ISSUE THREAD

**Issue by dehesselle** [dehesselle/chatty#7]
With throwing out JavaFX in Java 11, they also removed `java-packager` which is used to build the application bundle. That is the reason why we're still using Java 10.

The original roadmap scheduled the return of its functionality with a replacement tool, `jpackage` ([JEP 343](https://openjdk.java.net/jeps/343)), for JDK 13. However, it did not make it into the rampdown phase, so its arrival can be expected for JDK 14 at the earliest (no confirmation at the time of writing).

Updating Java is on hold until such a tool arrives.

**Comment by dehesselle** [comment]
JDK 14 has been released. The packaging tool `jpackage` is also part of OpenJDK. Looks like I can start looking into updating Java!

**Comment by dehesselle** [comment]
Initial test builds successful. This is shaping up to be a painless transition. 😄  
Haven't had one of those in a long time.

**Comment by dehesselle** [comment]
Fixed by https://github.com/dehesselle/chatty/commit/83434a01b2ab61661046e3b292090a1fe7b35d52
